server/app/models/grid_service.rb

```diff
@@ -147,7 +147,7 @@ def stack_exposed?
   end
 
   def deploy_pending?
-    self.grid_service_deploys.where(started_at: nil).count > 0
+    self.grid_service_deploys.where(finished_at: nil).count > 0
```

Comment:
> AFAIK this means that there could be any number of really old stale GridServiceDeploys that match this query, because AFAIK we don't clean out old/stale GridServiceDeploys?

These kind of stale deploys do seem to exist, and they don't appear to get cleaned out: `puts GridServiceDeploy.where(:finished_at => nil).asc(:created_at).map{|sd| "#{sd.created_at} #{sd.grid_service ? sd.grid_service.to_path : '<destroyed>'} is #{sd.deploy_state} with started_at=#{sd.started_at} finished_at=#{sd.finished_at}" }.join("\n")`
```
2016-11-22 09:18:20 UTC <destroyed> is created with started_at=2016-11-22T09:18:20+00:00 finished_at= 2017-02-24 13:37:58 UTC development/ingress-lb/lb is ongoing with started_at=2017-02-24T13:37:58+00:00 finished_at= 2017-03-03 09:19:53 UTC development/ingress-lb/lb is ongoing with started_at=2017-03-03T09:19:54+00:00 finished_at= 2017-04-03 07:23:40 UTC development/ingress-lb/lb is created with started_at=2017-04-03T07:23:40+00:00 finished_at= 2017-04-06 06:47:24 UTC development/ingress-lb/lb is created with started_at=2017-04-06T06:47:24+00:00 finished_at= 2017-04-11 07:56:23 UTC development/ingress-lb/lb is created with started_at=2017-04-11T07:56:24+00:00 finished_at= 2017-04-24 16:05:56 UTC development/null/redis-test is created with started_at=2017-04-24T16:05:56+00:00 finished_at= 2017-04-24 16:06:21 UTC development/null/redis-test is created with started_at=2017-04-24T16:10:59+00:00 finished_at=
```